docs/source/bigchaindb-cli.md

```diff
@@ -20,3 +21,65 @@ There's another command named `bigchaindb-benchmark`. It's used to run benchmark
 $ bigchaindb-benchmark -h
 $ bigchaindb-benchmark load -h
 ```
+
+Note that you can always start `bigchaindb` using a different config file using the `-c` option.
+For more information check the help with `bigchaindb -h`.
+
+
+# Precedence in reading configuration values
+
+Note that there is a precedence in reading configuration values:
+ - (**not yet**) command line;
+ - local config file;
+ - environment vars;
+ - default config file (contained in ``bigchaindb.__init__``).
+
+This means that if the default configuration contains an entry that is:
+
+```
+{...
+    "database": {"host": "localhost",
+                 "port": 28015}
+...}
+```
+
+while your local file `local.json` contains:
+```
+{...
+    "database": {"host": "whatever"}
+...}
+```
+
+and you run this command:
+```
+$ BIGCHAINDB_DATABASE_HOST=foobar BIGCHAINDB_DATABASE_PORT=42000 bigchaindb -c local.json show-config
+```
+
+you will get:
+```
+INFO:bigchaindb.config_utils:Configuration loaded from `local.json`
+{'CONFIGURED': True,
+ 'api_endpoint': 'http://localhost:8008/api/v1',
+ 'consensus_plugin': 'default',
+ 'database': {'host': 'localhost', 'name': 'bigchain', 'port': 28015},
+ 'keypair': {'private': 'Hbqvh...',
+             'public': '2Bi5NU...'},
+ 'keyring': [],
+ 'statsd': {'host': 'localhost', 'port': 8125, 'rate': 0.01}}
+
+```
+
+# List of env variables
+
+```
+- BIGCHAINDB_DATABASE_HOST
+- BIGCHAINDB_DATABASE_PORT
+- BIGCHAINDB_DATABASE_NAME
+- BIGCHAINDB_KEYPAIR_PUBLIC
+- BIGCHAINDB_KEYPAIR_PRIVATE
```

Comment:
> Need to add environment variables for the keyring (list of public keys of federation nodes). Maybe BIGCHAINDB_KEYRING?

done!

> Maybe also have an example of setting those values (e.g. in a .bashrc file), with typical/realistic values shown (i.e. a template)?

I think it's better if we create a new doc page describing all the config values, what do you think? we can do it after separating the client code from the server code.